api/src/opentrons/protocol_engine/state/module_substates/thermocycler_module_substate.py

```diff
@@ -170,7 +175,9 @@ def validate_ramp_rate(self, ramp_rate: float, target_temp: float) -> float:
                 f" while heating or {MAX_COOLING_RATE}°C/s when cooling."
             )
         if ramp_rate <= 0:
-            raise InvalidRampRateError("Thermocycler ramp rate cannot be negative or 0")
+            raise InvalidRampRateError(
+                f"Thermocycler ramp rate cannot be less than or equalt to 0, got {ramp_rate}"
```

Comment:
```suggestion
                f"Thermocycler ramp rate cannot be less than or equal to 0, got {ramp_rate}"
```
Probably shouldn't allow 0 here though.

Reply:
this does not allow 0